<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBConfig</class>
 <widget class="QDialog" name="DBConfig">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>260</width>
    <height>180</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>180</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>260</width>
    <height>180</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Base de Datos - Configuración</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Icons.qrc">
    <normaloff>:/icon/logo</normaloff>:/icon/logo</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="2">
    <widget class="QSpinBox" name="inputPort">
     <property name="maximum">
      <number>999999</number>
     </property>
     <property name="value">
      <number>3306</number>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="inputUser">
     <property name="placeholderText">
      <string>ej: root</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblUser">
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblPort">
     <property name="text">
      <string>Puerto</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblPassword">
     <property name="maximumSize">
      <size>
       <width>60</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Contraseña</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLineEdit" name="inputHostname">
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>ej: localhost</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QPushButton" name="btnSave">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancelar</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="btnDBTest">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Probar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="inputPassword">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>**************</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblHost">
     <property name="text">
      <string>Host</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputHostname</tabstop>
  <tabstop>inputPort</tabstop>
  <tabstop>inputUser</tabstop>
  <tabstop>inputPassword</tabstop>
  <tabstop>btnDBTest</tabstop>
  <tabstop>btnSave</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
